### ISSUE TITLE
Double-check render deployment

### ISSUE THREAD

**Issue by charlesBochet** [twentyhq/twenty#2033]
## Scope & Context

We have recently re-work our project installation to take into account postgres extension pg_graphql (needed for flexible backend). I fear this is breaking the render deployment. 
Investigate and fix problems

## Technical inputs

Try do deploy to render using the deployment button here: https://docs.twenty.com/hosting/self-hosting and make sure the project is running

**Comment by khakimov** [comment]
I’ve been testing on render recently, will try new version today 🙋‍♂️

**Comment by khakimov** [comment]
@charlesBochet you are right, render doesn't support pg_graphql ([list of supported extensions](https://render.com/docs/databases#postgresql-extensions))

```
driverError: error: extension "pg_graphql" is not available
detail: 'Could not open extension control file "/usr/pgsql-15/share/extension/pg_graphql.control": No such file or directory.'
```

another problem was metadata - need to call yarn database:setup to make sure metadata created. 

ps: I will look into supabase since they support [pg-graphql](https://supabase.com/blog/pg-graphql) for easier database deployment option.

**Comment by charlesBochet** [comment]
pg_graphql is core to our graphql approach. I am worried that this would actually disqualify most of the cloud providers. I thought that we had double check pg_graphql support on RDS but it seems it's not supported :(

**Comment by charlesBochet** [comment]
I'm writing to Supabase to get more information on how they approach the issue

**Comment by khakimov** [comment]
> I'm writing to Supabase to get more information on how they approach the issue

I already wrote an email to support@render.com asking if it is possible to activate pg_graphql. 

🥸 AWS nope [Extension versions for Amazon RDS for PostgreSQL](https://docs.aws.amazon.com/AmazonRDS/latest/PostgreSQLReleaseNotes/postgresql-extensions.html)

🙌 Vercel supports pg_graphql 1.1.0 [[Postgres Compatibility](https://vercel.com/docs/storage/vercel-postgres/supported-postgresql-extensions#)](https://vercel.com/docs/storage/vercel-postgres/supported-postgresql-extensions)

**Comment by charlesBochet** [comment]
Interesting, thanks a lot @khakimov!

Can we try to host postgres in a docker container and use the scripts to setup pg_graphql into that container on render? I think it should work!

**Comment by charlesBochet** [comment]
https://supabase.com/blog/pg-tle might also be a solution that we could try on rds

**Comment by khakimov** [comment]
@charlesBochet here is updated render.yaml and couple tweaks to make a docker container with psql + 
pg_graphql. 

<img width="1231" alt="image" src="https://github.com/twentyhq/twenty/assets/396223/9fbab4cd-607d-4266-a9ac-193f4067ba5d">

I will look into pg-tle, looks promising 👍

**Comment by charlesBochet** [comment]
Thank you! Let's close this issue!
